charts/linkerd-buoyant/templates/metrics-agent.yaml

```diff
@@ -18,7 +18,6 @@ data:
     metrics:
       wal_directory: /tmp/wal
       global:
-        scrape_interval: 10s
```

Comment:
Should we set that explicitly to avoid a situation where the default changes from 1m to something else?